OpenRA.Mods.Common/Traits/ExternalCapturable.cs

```diff
@@ -24,6 +24,9 @@ public class ExternalCapturableInfo : ITraitInfo
 		[Desc("Seconds it takes to change the owner.", "You might want to add a ExternalCapturableBar: trait, too.")]
 		public readonly int CaptureCompleteTime = 15;
 
+		[Desc("Whether autotargeting ignores this actor while it is being captured by an ally.")]
+		public readonly bool AutoTargetIgnores = false;
```

Comment:
I think you can safely set this to `true`. At least for the monster tank madness mission.